pom.xml

```diff
@@ -306,7 +306,7 @@
             <dependency>
               <groupId>com.puppycrawl.tools</groupId>
               <artifactId>checkstyle</artifactId>
-              <version>6.12.1</version>
+              <version>6.13</version>
```

Comment:
This can't prevent Apex core from upgrading. There are bug fixes included into the new version of the checkstyle and even if we don't hit those bugs in the current code line, we need those fixes to avoid introducing style violations into the future codeline.